pr2eus/robot-interface.l

```diff
@@ -1556,7 +1555,7 @@ send-action [ send message to action server, it means robot will move ]"
          (send msg :points (list pt1 pt2)))
        ))
      (send goal :goal :trajectory msg)
-     (when send-action and move-base-trajectory-action
```

Comment:
why this syntax error did not caught by test ???? https://github.com/jsk-ros-pkg/jsk_pr2eus/commit/af18a5a92ca13f328fac0931f4d50ebacb832efb